lib/devise/encryptable/encryptors/pbkdf2.rb

```diff
@@ -0,0 +1,16 @@
+module Devise
+  module Encryptable
+    module Encryptors
+      class Pbkdf2 < Base
+        def self.digest(password, stretches, salt, pepper)
+          pepper ||= ''
+          password ||= ''
+          digest = OpenSSL::Digest::SHA512.new
+          len = digest.digest_length
+          iterations = stretches > 1000 ? stretches : 100_000
```

Comment:
this seemed the safest way to make sure a sane number was used, if the user neglects to change their config when switching to this module.